x-pack/legacy/plugins/ml/public/data_frame/common/transform_list.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DataFrameTransformId, DataFrameTransformPivotConfig } from './transform';
+import { DataFrameTransformStats } from './transform_stats';
+
+// Used to pass on attribute names to table columns
+export enum DataFrameTransformListColumn {
```

Comment:
I know this isn't new code to this PR, but I wonder if we should be capitalizing all enum variable names?

Reply:
Fixed in 4e6a4cc.